fairlearn/metrics/_multi_curve_plot.py

```diff
@@ -0,0 +1,126 @@
+from ..utils._input_validation import _validate_and_reformat_labels, _validate_and_reformat_labels_and_sf, check_consistent_length
+from ..postprocessing._plotting import _MATPLOTLIB_IMPORT_ERROR_MESSAGE
+from ._make_derived_metric import _DerivedMetric
+from typing import Callable, List, Union
+
+def plot_model_comparison(
+    *,
+    x_axis_metric: Callable[..., Union[float, int]],
+    y_axis_metric: Callable[..., Union[float, int]],
+    y_true,
+    y_preds,
+    sensitive_features,
+    show_plot: bool,
+    ax=None,
+    **kwargs
```

Comment:
>     * I added a **kwargs to plot_model_comparison and forward this to Axes.scatter(x, y, **kwargs), is this even a good idea? It makes our function even more easy to use, but I did not find a good way to validate all the possible kwargs, and instead if an invalid kwarg is passed a not-super-informative error is given.

We discussed that in another PR that's adding a different plotting function and essentially reached the conclusion that forwarding the keyword args is a good idea. Users should not simply provide args they don't know and pass them through. We can always refine the error handling later on if it turns out to be required.